velox/exec/tests/utils/TpchQueryBuilder.cpp

```diff
@@ -153,6 +157,236 @@ TpchPlan TpchQueryBuilder::getQ1Plan() const {
   return context;
 }
 
+TpchPlan TpchQueryBuilder::getQ3Plan() const {
```

Comment:
On second thoughts, lets limit this PR to just getQ3Plan() where a consistent plan between duckdb and velox are used.

Please keep around the presto style plan around though. We could consider adding that as a separate test on the presto side later.